assets/src/edit-story/components/library/panes/media/common/mediaElement.js

```diff
@@ -250,6 +251,9 @@ const MediaElement = ({
     onInsert(resource, width, height);
   };
 
+  const showAttribution =
+    active && resource.attribution && resource.attribution.author;
```

Comment:
[nit] You can probably shorthand this to
`active && resource.attribution?.author`

Reply:
Done.